Google.PowerShell/Compute/GceCmdlet.cs

```diff
@@ -110,6 +122,51 @@ private void WriteWarnings(Operation op)
             }
         }
 
+        private void WriteProgress(Operation op, string progressMessage)
+        {
+            int activityId = (int)(op.Id % int.MaxValue ?? 0);
+            string activity = progressMessage ?? op.Description;
+            if (activity == null)
+            {
+                string operationType = op.OperationType;
+                operationType = operationType.First().ToString().ToUpper() + operationType.Substring(1);
+                int startIndex = op.TargetLink.IndexOf("projects", StringComparison.Ordinal);
+                if (startIndex < 0 || startIndex >= op.TargetLink.Length)
+                {
+                    startIndex = 0;
+                }
+                activity = operationType + " " + op.TargetLink.Substring(startIndex);
+            }
+            string statusDescription = op.StatusMessage ?? op.Status;
+            int percentComplete;
+            if (op.Progress == null || op.Progress == 0)
+            {
+                percentComplete = _spinnerValue = (_spinnerValue + 1) % 100;
```

Comment:
This seems kinda lame that the spinner is dependent upon how often `WriteProgress` is called. So different cmdlets will "progress" at different rates depending on arbitrarily chosen sleep intervals.

Of course, I can't think of anything better short of requiring that you explicitly start and stop the faux progress bar.